src/Settings.hs

```diff
@@ -114,3 +114,8 @@ programPath context@Context {..} = do
         install <- (\l -> l == stage || package == ghc) <$> maybeLatest
         let path = if install then installPath package else buildPath context
         return $ path -/- programName context <.> exe
+
+destDir :: Action FilePath
```

Comment:
I dropped the use of `env` in defining `DESTDIR` entirely. It can only customized through `UserSettings` now, but I think this is better.